src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -1,8 +1,9 @@
 package seedu.address.logic;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_MODULE_DISPLAYED_INDEX;
-import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+// import static seedu.address.commons.core.Messages.MESSAGE_INVALID_MODULE_DISPLAYED_INDEX;
+// import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
```

Comment:
Were these commented out because they were unused?

Reply:
Yes! If we don't, it'd fail the checkstyle.